src/Database/Statement/BufferedStatement.php

```diff
@@ -149,7 +149,7 @@ public function rowCount()
     {
         if (!$this->_allFetched) {
             $counter = $this->_counter;
-            while ($this->fetch('assoc')) {
+            while ($this->fetch(parent::FETCH_TYPE_ASSOC)) {
```

Comment:
These runtime ones should be static:: IMO.

Reply:
They should be.

Reply:
Thanks for you feedback today.